packages/flutter/lib/src/material/menu_bar_theme.dart

```diff
@@ -0,0 +1,333 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+import 'menu_bar.dart';
+import 'theme.dart';
+
+/// Defines the visual properties of [MenuBar], [MenuBarMenu] and
```

Comment:
This theme includes some properties that are specific to menu _bars_, but it seems to mostly be about menus. Maybe what we really need is MenuTheme with a few menu bar properties? Or maybe I've misunderstood what this theme is for; however I don't see a separate menu theme.

We need to explain how this theme relates to PopupMenus and PopupMenuTheme. Surely they're not completely different animals?